README.md

```diff
@@ -12,77 +12,73 @@ INSTALLED_APPS = [
 ] + get_core_apps([
     "oscar_elasticsearch.search", # replace standard oscar app
 ]) + [
+    "widget_tweaks",
     "wagtail.core",
-    "wagtail.contrib.search_promotions", # allow search result display and such
     "wagtail.search",
```

Comment:
Will we remove other wagtail apps as well?

Reply:
Hmm maybe we can remove wagtail.core, I think the wagtail.search has to stay because of the query stats model

Reply:
I tried, it's not possible in the current situation